index.rst

```diff
@@ -1,57 +1,435 @@
-..
-  Technote content.
+:tocdepth: 1
 
-  See https://developer.lsst.io/restructuredtext/style.html
-  for a guide to reStructuredText writing.
+.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+.. sectnum::
 
-  Use the following syntax for sections:
+.. _introduction:
 
-  Sections
-  ========
+Introduction
+============
 
-  and
+This tech note describes the uses of Data Management (DM) Science Pipelines code in observatory operations, including the computing and storage systems available to it as Commissioning proceeds.
 
-  Subsections
-  -----------
+.. _use-cases:
 
-  and
+Use Cases
+=========
 
-  Subsubsections
-  ^^^^^^^^^^^^^^
+DM Science Pipelines code has always been expected to produce Prompt data products and telemetry feedback to the Observatory Control System.
+But it is now expected to be present in several other operational systems to perform:
 
-  To add images, add the image file (png, svg or jpeg preferred) to the
-  _static/ directory. The reST syntax for adding the image is
+* ISR of wavefront sensor images for the active optics system (AOS).
+* Processing of full-focal-plane intra-focal and extra-focal images for the AOS.
+* Centroiding of guider images.
+* Initial spectrum processing to control the LSST Auxiliary Telescope Imager and Slitless Spectrograph (LATISS) positioning and focus.
+* Automatic rapid analysis and visualization of images in the Camera Diagnostic Cluster.
+* Ad hoc and scripted Commissioning-related analyses of images and telemetry.
 
-  .. figure:: /_static/filename.ext
-     :name: fig-label
+In addition, the Science Pipelines environment is used as the basis for Telescope and Site Software (TSSW) Commandable SAL Component (CSC) development.
 
-     Caption text.
+The `next section <satisfying-use-cases>`_ describes how these use cases are handled.
+The Commissioning use case, because of its complexity and significance is described in its `own section <commissioning-analytics>`_.
+The `following sections <summit-resources>` give more details about the services and resources available to execute the use cases.
```

Comment:
Missing an underscore.

Reply:
No, worse than that — all of the links are broken, it's just that this one was obvious. You need to use `:ref:`, I imagine.